### ISSUE TITLE
Killer example: family gathering, birthday party

### ISSUE THREAD

**Issue by pdehaye** [CrowdNotifier/documents#18]
As mentioned in #8, a physically printed QR-code is a bit of a red herring. The protocol itself is interesting regardless of how it will be instantiated. I think stepping away from a physically printed QR-code opens up the range of events/settings that should be considered, and in some cases it would be a real need (as opposed to substituting to solutions that have been put in place over past few weeks in Europe that do work, and for which it will be hard to make an argument around privacy). 

In other words, make sure to explicitly include the following events as well:
* extended family gathering (such as religious celebrations like a family Christmas party)
* birthday party

**Comment by pdehaye** [comment]
Announced today: https://www.20min.ch/fr/story/repas-entre-amis-letat-devra-etre-prevenu-964835255053

**Comment by wouterl** [comment]
These, and other such events, are captured by our broad definition of what constitutes a location.

We note, however, that digital contact and presence tracing solutions are useful because they identify contacts that cannot easily be determined through manual contact tracing. The examples provided in this issue typically do not share this property. Attendees to either can usually be quickly  identified.

**Comment by pdehaye** [comment]
Have you talked to contact tracers? I think part of the utility of CrowdNotifier would be derived from "identifying"/"naming" (in the sense of "assigning an identifier") more fluid situations where co-presence cannot easily be determined through manual contact tracing precisely due to the lack of a standard way of naming "events" (or as indicated in #8, "contexts"). 

I know it doesn't sound like a real problem, but imagine: x cases a day come in, through y separate individuals working in the tracing unit. How do you know that two cases' events are actually the same event? Defining those situations does not quite map to location (maybe it was in a big building with many events?) or the activity (what is the choral practice for one will be the orchestra rehearsal for the other, etc; parent only knows birthday child's first name for a birthday, etc). 

Additionally, you are making assumptions on the fact that what is currently done for contact tracing will remain the same, while evidence indicates otherwise (cue the shift to venue-based tracing in the first place).
